.github/ISSUE_TEMPLATE/hackathon-project-template.md

```diff
@@ -1,9 +1,96 @@
 ---
 name: Brainhack Project Template
-about: Project submissions are now closed!
+about: This provides the backbone structure for your project description. Please follow along to submit your project!
 title: ''
 labels: 'Hackathon project'
 assignees: '@ohbm/project-monitors'
```

Comment:
Before having this up&running we need to set up the project-monitor team, the one from last year has been renamed [hack2020-monitors](https://github.com/orgs/ohbm/teams/hack2020-monitors)

Reply:
Sure. But would it influence this PR?

Reply:
It will probably make the repo misbehave as GitHub would try to assign any future issue to the team "@ohbm/project-monitors" that at the moment does not exist.

Reply:
Should I comment it out for now?

Reply:
I created the team "@ohbm/hack2021-monitors" (https://github.com/orgs/ohbm/teams/hack2021-monitors/members), so you can update the assignee field with that.